net/unbound/files/unbound.sh

```diff
@@ -817,15 +979,16 @@ unbound_uci() {
   config_get UNBOUND_TTL_MIN     "$cfg" ttl_min 120
   config_get UNBOUND_TXT_DOMAIN  "$cfg" domain lan
 
+  UNBOUND_LIST_DOMAINS="nowhere $UNBOUND_TXT_DOMAIN"
```

Comment:
Guarantees there is some text in the aggregated domain list variable. "nowhere" should never be a TLD.